python/src/spark_rapids_ml/classification.py

```diff
@@ -979,9 +979,35 @@ def _logistic_regression_fit(
             if self.getStandardization() is True and is_sparse is True:
                 concated = concated.toarray()
 
+            num_pad_zero_cols = 0
```

Comment:
Maybe add a comment about why this is done and that it is temporary work around.

Reply:
Sounds good. Added a comment.